test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -195,30 +195,22 @@ Describe "Deserializing corrupted Cim classes should not instantiate non-Cim typ
 
         # Only run on Windows platform.
         # Ensure calc.exe is avaiable for test.
-        if ( !$IsWindows -or ((Get-Command calc.exe 2>$null) -eq $null) )
-        {
-            $orginalDefaultParameters = $PSDefaultParameterValues.Clone()
-            $PSDefaultParameterValues["it:skip"] = $true
-        }
-        else
+        $shouldRunTest = $IsWindows -or ((Get-Command calc.exe -ea SilentlyContinue) -ne $null)
+        $skipNotWindows = ! $shouldRunTest
```

Comment:
i changed it to this way since there was only a single test. There's a couple of patterns, if you have a large number of tests you want to mark skipped, then PSDefaultParameterValues make some sense, but with a small number this can be easier